### DISCUSSION TITLE
Elements?

### DISCUSSION THREAD

**Discussion by ghost** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1230] [General]
I have an idea to add the periodic table like in Minecraft education.

first you have a lifting machine which needs a lot of power to make elements, you need one more element like two water buckets, you can make one tank of helium

second, you have a machine that can downgrade elements like hydrogen -> helium

With those machines you can make blocks such as iron or gold, but you cannot make schrabidium with them.

**Comment by JameH2** [comment]
and just like Minecraft education edition, i believe you should go back to school

**Reply by ghost** [reply]
shiver my timbers
![image](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/assets/131010857/f90b46f1-72f2-44bb-890a-4e8afb659f16)
